android/src/com/frostwire/android/gui/transfers/TransferManager.java

```diff
@@ -303,7 +303,9 @@ public boolean remove(Transfer transfer) {
 
     public void pauseTorrents() {
         for (BittorrentDownload d : bittorrentDownloads) {
-            d.pause();
+            if (!d.isSeeding() && d.isDownloading()) {
```

Comment:
will have to rename this method to avoid confusion -> `pauseDownloadingTorrents()`